README.md

```diff
@@ -82,6 +84,7 @@ OpenTelemetry can collect tracing data automatically using instrumentations. Ven
 - [@opentelemetry/instrumentation-xml-http-request][otel-instrumentation-xml-http-request]
 - [@opentelemetry/instrumentation-fetch][otel-instrumentation-fetch]
 - [@opentelemetry/instrumentation-document-load][otel-contrib-instrumentation-document-load]
+- [@opentelemetry/instrumentation-long-task][otel-contrib-instrumentation-long-task]
 - [@opentelemetry/instrumentation-user-interaction][otel-contrib-instrumentation-user-interaction]
 - [@opentelemetry/plugin-react-load][otel-contrib-plugin-react-load]
```

Comment:
The node list is sorted alphbetically.
If you have motivation, you can sort this list as well ;)

Reply:
Sorted! https://github.com/open-telemetry/opentelemetry-js-contrib/pull/827/commits/b6e18f93ea37e4f125a4df1d33511b0e9fb2e5cb